### PR TITLE
Switch to influxdb for reporting metrics.

### PR DESCRIPTION
**Pull request by danielimre**

### CODE REVIEW

File: docker/redis/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: "3.7"
+
+services:
+  redis:
+    image: redis
```
Comment (bejancsaba) [inline docker/redis/docker-compose.yml:5]: What is this used for currently?

File: mirage/pom.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
   <description>WebService with some tricks</description>
 
   <properties>
-    <java.version>13</java.version>
+    <java.version>14</java.version>
```
Comment (bejancsaba) [inline mirage/pom.xml:19]: Not F@cking around :)

File: docker/monitoring/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+
+services:
+  zipkin:
```
Comment (bejancsaba) [inline docker/monitoring/docker-compose.yml:4]: What do you think about defining the storage type, could be persisted to Cassandra (could be useful if Zipkin is shut down for any reason)

**Review by bejancsaba — DISMISSED** [review]
Just minor thoughts

**Review by bejancsaba — COMMENTED** [review]
One more thing I just realized that readme needs to be updated too. docker-compose-monitoring.yml was moved and renamed and also the grafana data source changed to influx, we need to reflect these changes there too.

File: docker/monitoring/docker-compose.yml (hunk 3 above)
Comment (danielimre) [reply docker/monitoring/docker-compose.yml:4]: I see no reason to persist traces.

Comment (bejancsaba) [reply docker/monitoring/docker-compose.yml:4]: I don't have strong feelings about this either way. So I'm ok with it :)

**Review by bejancsaba — DISMISSED** [review]

File: docker/redis/docker-compose.yml (hunk 1 above)
Comment (danielimre) [reply docker/redis/docker-compose.yml:5]: Nothing.

**Pull request merged**